config.js

```diff
@@ -4,14 +4,21 @@ function getConfig(env) {
     switch (env) {
 
     case 'production':
-    case 'development':
         return {
-            networkId: 'default',
+            networkId: 'mainnet',
             nodeUrl: 'https://rpc.nearprotocol.com',
             contractName: CONTRACT_NAME,
             walletUrl: 'https://wallet.nearprotocol.com',
             helperUrl: 'https://helper.nearprotocol.com',
         };
+    case 'development':
+        return {
+            networkId: 'testnet',
```

Comment:
maybe makes sense to keep this as `default` to avoid losing keys (though alternatively we can document how to move keys).

Reply:
I reverted my change